docs/pelion_ready/introduction_chip2cloud.md

```diff
@@ -4,7 +4,7 @@ You can connect Mbed OS devices to the Pelion Iot Platform and use [the full sui
 
 The documentation covers:
 
-1. [Using Pelion Ready boards](../mbed-os-pelion/index.html) to establish a connection to the platform.
+1. [Using Pelion Ready boards](../mbed-os-pelion/device-management-for-mbed-os.html) to establish a connection to the platform.
```

Comment:
Not entirely sure where the link for the Pelion Ready boards should lead.

Reply:
Exactly there; it's just not called Pelion Ready externally

Reply:
What should we call them? Mbed enabled?

Reply:
Sorry, didn't mean "externally", meant the page title is about DM. The name here is fine, and so is the link.